cli/command/command.go

```diff
@@ -144,6 +144,18 @@ func ConvertOpenShiftCommand() cli.Command {
 				Name:  "deploymentconfig,dc",
 				Usage: "Generate a OpenShift DeploymentConfig object",
 			},
+			cli.StringFlag{
```

Comment:
But all those things can be set by modifying git repository.
To set right remote you can update repository remote using `git remote`. And same thing for branch, you just switch git branch.